src/app/intermission/intermission.html

```diff
@@ -8,6 +8,6 @@
                 <img src="../assets/logo.png" width="300px"/>
             </div>
         </div>
-        <div class="title">Randomizer Tournament 2018</div>
+        <div class="title">All Skills Tournament 2018</div>
     </div>
 </div>
```

Comment:
Let's get rid of IntermissionComponent too, for now, unless you think it could be useful for you.